tests/suite/cli_misc.rs

```diff
@@ -713,6 +713,45 @@ info: toolchain 'test' uninstalled
     });
 }
 
+// issue #3737
+#[test]
+#[cfg(any(unix, windows))]
+fn toolchain_symlink() {
```

Comment:
```suggestion
/// the ~/.rustup/toolchains path is permitted to be a symlink
fn toolchains_symlink() {
```

Reply:
I want to do this as well, however I put this function right below

https://github.com/rust-lang/rustup/blob/29ce9ee0ae3e5f12e217d939f6e82c05fbf0d4dd/tests/suite/cli_misc.rs#L680-L683

for a reason. Maybe I should change the name of that test accordingly.